### ISSUE TITLE
Message to tell users that they can't download images directly from GBIF

### ISSUE THREAD

**Issue by ManonGros** [gbif/portal-feedback#1717]
It would be good to let the users know that they can't download the images from the gallery directly from GBIF. Perhaps with a message in the email that we send for the downloads? What do you think?

Something like that?
```
If you wish to access the images displayed in the gallery, please use the URL provided.
Be aware that licenses applied to images may differ from those applied to occurrence records.
```

Following some feedback we got from a user (check helpdesk emails 03/01/2019).

**Comment by dnoesgaard** [comment]
@ManonGros can you provide the user feedback here for context, please?

**Comment by ManonGros** [comment]
The first email we got had the following message:

> I am trying to download a large number of images from GBIF. I can get the query and down load a csv or download teh same data using the rgbif or spocc packages in R. However, I don't know how to get the actual images. I have tried downloading a Darwin Core Archive but at 3mb, this seems far to small to contain 4000 images and I don't understand how to use it anyway.
>  
> I can view images using rgbif but can't actually download them as jpegs.
>  
> Is there some sort of guide to downloading images, Or am i supposed to just get the URLs and download them off the sites directly?

I suggested to use the URLs to download the images and the user mentioned this in his reply:

> It might be helpful to make it more explicit that the photos are downloaded using the link, this was a bit confusing as some sites don't like being pinged for thousands of downloads from a single IP address.

**Comment by MattBlissett** [comment]
In reality, many publishers won't like users downloading large numbers of images directly from the publisher.

We're going to work on image download formats later this year, but there will be licensing issues for us to consider.

**Comment by dnoesgaard** [comment]
Do we have an (un)official policy on downloading images via the cache?

**Comment by MortenHofft** [comment]
Would it make sense to have it as part of the download format description? Say in the popup for csv/dwc or in that description? 
`Tab delimited CSV. Only contains the dataafter GBIF interpretation (multimedia not included) (?)`

**Comment by dnoesgaard** [comment]
I did include a line in the DwC pop-up:

> Additional files provide supplementary information such as links to images (but not the actual image files).

My rationale for adding it elsewhere was to avoid duplication, as neither option includes any image files.

**Comment by MortenHofft** [comment]
Makes sense, but more download formats/buttons are coming (species download, ebv, images?)

**Comment by dnoesgaard** [comment]
Ah cool, I see. Perhaps it would make sense to have a table at some point clearly showing differences, e.g.

|  | CSV | DwC | Image archive | EBV | etc. |
|-----------------------------|-----|-----|---------------|-----|------|
| Raw data |  | ✓ | x | x | x |
| Interpreted occurrence data | ✓ | x | x | x | x |
| Multimedia files | x | x | ✓ | x | x |
| Etc. |  | |  |  |  |

**Comment by dnoesgaard** [comment]
Morten what about having something in the gallery, e.g

<img width="774" alt="screenshot 2019-01-21 at 14 54 12" src="https://user-images.githubusercontent.com/21358085/51478695-78279e00-1d8c-11e9-828e-565edce51c22.png">

linking to the [pop-up](https://www.gbif.org/faq?question=how-do-i-download-images-shown-in-the-gallery)?

**Comment by MortenHofft** [comment]
I like that suggestion. That reminds we that we used to have some text on the top of images. I wonder what that was.

**Comment by ManonGros** [comment]
There is a message that appear when filtering for licenses:
![screenshot 2019-01-22 at 10 00 15](https://user-images.githubusercontent.com/7677271/51523738-980e9e80-1e2c-11e9-9be0-94dd9c766ba4.png)

**Comment by SenCodeMaker** [comment]
hello everyone ! in continuation of this question, is there a way to filter by image license ?

**Comment by ManonGros** [comment]
Hi @SenCodeMaker there is no way to filter images by license GBIF (as they are not standardised). That being said, if you have a specific use case, you can write to helpdesk@gbif.org to ask for a custom download.
